src/styles/main.scss

```diff
@@ -44,8 +44,27 @@ body {
 
         &--color {
             display: flex;
-            max-width: 1140px;
+            flex-wrap: wrap;
             margin: 0 auto;
+            padding-left: 15px;
+            padding-right: 15px;
+
+            @media (min-width: 576px) {
+                width: 540px;
+                max-width: 100%;
+            }
+            @media (min-width: 768px) {
+                width: 720px;
+                max-width: 100%;
+            }
+            @media (min-width: 992px) {
```

Comment:
There already is `$screen-md-min: 1024px;` on the top of the file so I’d try to reuse that var instead in order to keep the number of breakpoints as small as possible.

Then I’d add other breakpoints (576, 768, 1200) as variables as well so it’s more transparent which ones should be used.